recipes/cms-fireworks/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "fwlite" %}
+{% set version = "10.5.0" %}
+{% set hash = "2f5c169b6ad54330f8c8ffee8ac287c719b579c2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/gartung/fwlite.git
+  git_rev: {{ hash }}
+  folder: src
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - git
+    # Required for OpenGL support
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cdt#libgl
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+  host:
+    - boost =1.68.0
+    - clhep =2.4.0.2
+    - cms-md5
+    - hepmc2
+    - libsigcpp
+    - libuuid
+    # - openssl
+    - pybind11
+    - python
+    - root
+    - tbb
+    - tinyxml2
+    - xerces-c
+    - xorg-libx11  # [linux]
+    - xorg-libxau  # [linux]
+    - xorg-libxcursor  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libxfixes  # [linux]
+    - xorg-libxft  # [linux]
+    - xorg-libxpm  # [linux]
+    - xz
+    - zlib
+  run:
+    - {{ pin_compatible('boost', max_pin='x.x') }}
+    - {{ pin_compatible('clhep', max_pin='x.x.x.x') }}
+    - cms-md5
+    - hepmc2
+    - libsigcpp
+    # - openssl
+    - python
+    - tbb
+    - tinyxml2
+    - xorg-libx11  # [linux]
+    - xorg-libxau  # [linux]
+    - xorg-libxcursor  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libxft  # [linux]
+    - xorg-libxpm  # [linux]
+    - xz
+    - zlib
+
+test:
+    commands:
+      - ldd ${CONDA_PREFIX}/bin/cmsRun  # [linux]
```

Comment:
Are you able to add any tests here that check if the binaries work? Or is everything visual?

I guess you'd like to change this to `cmsShow.exe` at the very least?

Reply:
It is all visual.